gdal/port/cpl_conv.h

```diff
@@ -281,6 +281,11 @@ char* CPLsetlocale (int category, const char* locale);
 void CPLCleanupSetlocaleMutex(void);
 /*! @endcond */
 
+/* -------------------------------------------------------------------- */
+/*                          CPLIsPowerOfTwo()                           */
+/* -------------------------------------------------------------------- */
+bool CPL_DLL CPLIsPowerOfTwo( unsigned int i );
```

Comment:
We try to stick to C89, so either return int, or put it in the C++ specific part of the header file (which is OK as the intended usage is rather internal)